Makefile.am

```diff
@@ -20,9 +20,6 @@ dist_noinst_DATA = logrotate.spec
 
 EXTRA_DIST = ChangeLog.md README.md autogen.sh examples
 
-# deprecated hard-wired Makefile, will be removed eventually
-EXTRA_DIST += Makefile.legacy
-
```

Comment:
When you remove it from distribution tarballs, it makes no sense to keep the file in the git repository.  Whoever needs the file can checkout it from an older commit when he/she already has (a deep enough) clone of the git repository.